tests/Tasks/UpdatePackageInfoTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace BringYourOwnIdeas\Maintenance\Tests\Tasks;
+
+use BringYourOwnIdeas\Maintenance\Util\ComposerLoader;
+use SapphireTest;
+use BringYourOwnIdeas\Maintenance\Tasks\UpdatePackageInfo;
+
+class UpdatePackageInfoTest extends SapphireTest
+{
+    public function testGetPackageInfo()
+    {
+        $loader = $this->createMock(ComposerLoader::class);
```

Comment:
createMock is available in PHPUnit 5 but SilverStripe 3 uses 3.7 by default. Can you please switch to `$this->getMockBuilder(ComposerLoader::class)->setMethods(['getLock'])->getMock()` which is B/C